src/site/filters/liquid.js

```diff
@@ -390,6 +390,16 @@ module.exports = function registerFilters() {
     return fieldLink;
   };
 
+  liquid.filters.deriveLastBreadcrumbFromPath = (breadcrumbs, string) => {
+    const last = {
+      url: { path: breadcrumbs.path, routed: true },
```

Comment:
Just to make sure, but `breadcrumbs` is an array, so does it have this `.path` property tied to it? Isn't that located on the parent property `entityUrl`, at `entityUrl.path`?

Reply:
That path property does exist, @ncksllvn, but I'm digging into the breadcrumb.url path property.
![image](https://user-images.githubusercontent.com/2404547/92753005-7347a180-f357-11ea-88db-ce44713ed422.png)

Reply:
Updated, @ncksllvn